compiler/luci/pass/include/luci/Pass/ConvertNCHWToNHWCPass.h

```diff
@@ -35,11 +35,22 @@ namespace luci
  */
 struct ConvertNCHWToNHWCPass final : public logo::Pass
 {
+public:
+  ConvertNCHWToNHWCPass(bool preserve_input = false, bool preserve_output = false)
```

Comment:
We usually don't use default argument values. please remove `= false` for `preserve_output`

Reply:
Can I ask you why we don't use default arguments?

Default arguments can reduce the changes in the test codes. I can remove them in this PR, but am curious about the reason.

Reply:
Cause of code is implicit and implicit codes may make problems when there are more changes later. this was from when @parjong started.

Reply:
I see. Thanks :)